vector/src/main/java/im/vector/app/features/home/room/filtered/FilteredRoomFooterItem.kt

```diff
@@ -33,11 +34,21 @@ abstract class FilteredRoomFooterItem : VectorEpoxyModel<FilteredRoomFooterItem.
     @EpoxyAttribute
     var currentFilter: String = ""
 
+    @EpoxyAttribute
+    var inSpace: Boolean = false
+
     override fun bind(holder: Holder) {
         super.bind(holder)
         holder.createRoomButton.onClick { listener?.createRoom(currentFilter) }
         holder.createDirectChat.onClick { listener?.createDirectChat() }
         holder.openRoomDirectory.onClick { listener?.openRoomDirectory(currentFilter) }
+
+        holder.openRoomDirectory.setText(
+                if (inSpace) R.string.space_explore_activity_title else R.string.room_filtering_footer_open_room_directory
```

Comment:
Q?: if we're always setting the text in the holder, should we also remove text from the layout? https://github.com/vector-im/element-android/pull/4124/files#diff-957e5d9987bce3ce857ba9f879945eab7131bc80e513432216e240e307aabf71R41

Reply:
Your link is a self limk, you mean https://github.com/vector-im/element-android/blob/develop/vector/src/main/res/layout/item_room_filter_footer.xml#L41

Would be better to replace `android:text` by `tools:text` (and format)

Reply:
argh it only works if the file is expanded :facepalm:

Reply:
I did the change in f6b672546791e56e9618e2b6ba195bd202b48320, @ouchadam, if it's OK for you, you can approve and auto-merge this PR